crowbar_framework/app/models/node_object.rb

```diff
@@ -1210,8 +1224,11 @@ def set_state(state)
       rescue Timeout::Error
         Rails.logger.warn("chef client seems to be still running after 5 minutes of wait; going on with the reboot")
       end
-
-      ssh_cmd("/sbin/reboot")
+      if @node[:platform] == "windows"
+        bmc_cmd("power cycle")
```

Comment:
@ovisan yes, there would be a similar `case` as in `bmc_cmd`, but there would not be any checks for platform.